Tests/test_SeqIO_StructureIO.py

```diff
@@ -0,0 +1,185 @@
+# Copyright 2012 by Eric Talevich.  All rights reserved.
+# This code is part of the Biopython distribution and governed by its
+# license.  Please see the LICENSE file that should have been included
+# as part of this package.
+
+import unittest
+import warnings
+
+try:
+    import numpy
+    from numpy import dot  # Missing on PyPy's micronumpy
+    del dot
+    # We don't need this (?) but Bio.PDB imports it automatically :(
+    from numpy.linalg import svd, det  # Missing in PyPy 2.0 numpypy
+except ImportError:
+    from Bio import MissingPythonDependencyError
+    raise MissingPythonDependencyError(
+        "Install NumPy if you want to use PDB formats with SeqIO.")
+
+from Bio import SeqIO
+from Bio import BiopythonWarning
+from Bio.PDB.PDBExceptions import PDBConstructionWarning
+
+
+def SeqresTestGenerator(extension, parser):
+    """
+    Test factory for tests reading SEQRES (or similar) records.
```

Comment:
Please remove the new line to follow PEP257 here
https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings